ci/management/commands/generate_reports.py

```diff
@@ -141,14 +152,19 @@ def get_identity(self, ids_client, identity):
         self.identity_cache[identity] = identity_object
         return identity_object
 
+    def get_registrations(self, hub_client, **kwargs):
+        registrations = hub_client.get_registrations(kwargs)
+        cursor = registrations['next']
+        while cursor:
+            yield registrations
```

Comment:
If we do a
```
for r in registrations.get('results', []):
    yield r
```
over here, that will elimate the need for the double loop down below